wordcloud/wordcloud.py

```diff
@@ -429,7 +429,7 @@ def generate_from_frequencies(self, frequencies, max_font_size=None):
             tried_other_orientation = False
             while True:
                 # try to find a position
-                font = ImageFont.truetype(self.font_path, font_size)
```

Comment:
thanks for the pull request. Can you please fix the source of the error? I think it's line 410, replace the ``/`` with a ``//``. You're on Python2.7, right?